ms-internal/git-merge.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
```

Comment:
Suggestion: Add bash flags to catch exit != 0 so the script fails. There are multiple calls to git for instance that can go wrong.

Reply:
we think the .sh fails 'appropriately' without that change -- but open to discuss -- from all our testing, since it's the `git push` failure that we're most interested in

Reply:
fixed with @juzuluag watching